lib/global.gd

```diff
@@ -196,18 +196,21 @@ DeclareGlobalFunction("MakeConstantGlobal");
 ##  <#GAPDoc Label="BindGlobal">
 ##  <ManSection>
 ##  <Func Name="BindGlobal" Arg='name, val'/>
+##  <Func Name="BindConstant" Arg='name, val'/>
 ##
 ##  <Description>
-##  sets the global variable named by the string <A>name</A> to the value
-##  <A>val</A>, provided it is writable, and makes it read-only.
+##  <Ref Func="BindGlobal"/> and <Ref Func="BindConstant"/> set the global
+##  variable named by the string <A>name</A> to the value
+##  <A>val</A>, provided it is writable. <Ref Func="BindGlobal"/> makes
```

Comment:
The "it" in "it is writable" is ambiguous. Perhaps "provided that variable is writable", or even "provided that variables is unbound, or bound and writable".

[Aha, but I just realize the ambiguous text was there before. *sigh*]